modules/node_modules/@ncigdc/routes/CartRoute.js

```diff
@@ -34,6 +35,15 @@ export default connect(state => (state.cart))(
       Container={CartPage}
       queryConfig={new CartRoute(props)}
       environment={Relay.Store}
+      render={({ error, props }) => {
```

Comment:
and to answer your question.. as long as the filetable triggers its own query via `setVariables` and `@include` then it can be in its own Loader and the summary cards query can return first.